Update `update_and_migration/updating.md` to add instructions for Meilisearch Cloud (#2282)
meilisearch/documentation: learn/update_and_migration/updating.mdx

This guide only works for versions v0.15 and above. If you are using an older version, please [contact support](https://discord.gg/meilisearch) for more information.
</Capsule>

## Updating on Meilisearch Cloud

> If you're using Meilisearch Cloud, you don't need to follow the rest of this guide. Clicking the "Update to `vX.Y.Z`" will take care of the update for you.

Reviewer: ```suggestion
If you're using Meilisearch Cloud, you don't need to follow the rest of this guide. Clicking the "Update to `vX.Y.Z`" button in the dashboard will take care of the update for you.
```

> Once a new version of Meilisearch is available, you should see it next to the "Meilisearch version" field under "Project overview".

Reviewer: ```suggestion
Click on the project you want to update. Look for the "Project overview" section at the top of the page. Once a new version of Meilisearch is available, you should see an update button next to the "Meilisearch version" field.
```

![Button to update Meilisearch version to 1.0.2](https://raw.githubusercontent.com/meilisearch/documentation/new-docs-content-migration/assets/images/updating/update-button.png)

> Clicking the button will take you to a pop up informing you that the update may take time. You cannot send any updates to the project but will still be able to continue searches during the update.
> 
> ![Pop up for confirming update to latest version](https://raw.githubusercontent.com/meilisearch/documentation/new-docs-content-migration/assets/images/updating/pop-up.png)
> 
> Once you agree to the update, the "Status" changes to "updating".

Reviewer: Not necessary to rewrite the pop-up, they can read it there.

```suggestion
Clicking the button will open a pop-up warning. Once you agree to the update, the "Status" of your project will change from "running" to "updating".
```

![Project update in progress](https://raw.githubusercontent.com/meilisearch/documentation/new-docs-content-migration/assets/images/updating/update-in-progress.png)

> On successfully updating the project, you will receive an email confirming the update and the "Status" will change to "running".

Reviewer: ```suggestion
Once the project has been successfully updated, you will receive an email confirming the update and the "Status" will change back to "running".
```
